.travis.yml

```diff
@@ -2,6 +2,7 @@
 language: python
 
 python:
+- "2.7"
 - "3.6"
```

Comment:
By pulling out the `f` string and `yield from`, this should make it compatible with anything above 2.7. It'd be a good idea to add support for 3.0 - 3.5 as well.